### PR TITLE
Use interactive `c' instead of `s'

### PR DESCRIPTION
**Pull request by zonuexe**
This change make similar operability of Vim.

### CODE REVIEW

File: ciel.el [hunk 1]
```diff
@@ -83,7 +83,8 @@
 ;;;###autoload
 (defun ciel-ci (arg)
   ""
-  (interactive "sci: ")
+  (interactive "cci: ")
+  (when (integerp arg) (setq arg (char-to-string arg)))
   (let ((%region))
     (cond ((or (string= arg "(") (string= arg ")")) (setq %region (region-paren "(")))
 	  ((or (string= arg "[") (string= arg "]")) (setq %region (region-paren "[")))
```
Comment (zonuexe) [inline ciel.el:90]: You can also be written as follows:

``` el
(cond ((or (eq arg ?\() (eq arg ?\))) (setq %region (region-paren "(")))
      ((or (eq arg ?\[) (eq arg ?\])) (setq %region (region-paren "[")))
      ((or (eq arg ?\{) (eq arg ?\})) (setq %region (region-paren "{")))
      ((or (eq arg ?\")
           (eq arg ?\')
           (eq arg ?\`))
       (setq %region (region-quote arg)))
      ((eq arg ?w) (setq %region (region-word))))
```

File: ciel.el [hunk 2]
```diff
@@ -83,7 +83,8 @@
 ;;;###autoload
 (defun ciel-ci (arg)
   ""
-  (interactive "sci: ")
+  (interactive "cci: ")
+  (when (integerp arg) (setq arg (char-to-string arg)))
```
Comment (zonuexe) [inline ciel.el:87]: This is unnecessary conversion

File: ciel.el [hunk 3]
```diff
@@ -102,7 +103,8 @@
 ;;;###autoload
 (defun ciel-co (arg)
   "COpy inside."
-  (interactive "sco: ")
+  (interactive "cco: ")
```
Comment (zonuexe) [inline ciel.el:106]: CCO… Me quemo!

**Comment by cs14095** [comment]
Thank you for requesting.
I completely forget to chage it.
It was preparation to kill between different character (see features) , but now I'm not gonna add the function. 
Anyway, your request is right.
I'll merge it tonight.

**Comment by zonuexe** [comment]
Thanks!

File: ciel.el (hunk 1 above)
Comment (cs14095) [inline ciel.el:90]: I have no idea but this doesn't work.
`string=` is ok.

**Pull request merged**